pkg/operator/ceph/csi/controller.go

```diff
@@ -133,10 +133,44 @@ func (r *ReconcileCSI) Reconcile(context context.Context, request reconcile.Requ
 // allow overriding for unit tests
 var reconcileSaveCSIDriverOptions = SaveCSIDriverOptions
 
+func (r *ReconcileCSI) csiDriverDisabled() bool {
```

Comment:
we dont really need helper function to get one value, please directly parse from the configmap when required, the same way we are doing for other variables in reconcile function